Code/C++/rat_in_maze.cpp

```diff
@@ -95,9 +78,29 @@ int main()
     return 0;
 }
 
-
-
-
+/* TestCase 1:
+	Input:
+		N = 3
+		maze[3][3] = 1 0 1
+			     1 0 1
+			     1 1 1
+	Output:
+		1 0 0 
+		1 0 0 
+		1 1 1 
+   TestCase 2:
+	Input:
+		N  = 4
+		maze[4][4] =  1 0 0 1
+			      1 1 1 0
+			      1 0 1 1
+			      0 0 1 1
+	Output:
+		1 0 0 0
+                1 1 1 0
+                0 0 1 0
+		0 0 1 1
+*/
 // Time Complexity = O(2^n)
```

Comment:
add the complexities in the above multi-line comments.

Reply:
ok Done mam.